packages/dd-trace/test/log.spec.js

```diff
@@ -278,4 +278,18 @@ describe('log', () => {
       expect(console.error).to.have.been.calledOnce
     })
   })
+
+  describe('unsubscribe', () => {
+    it('should fail when invoking unsubscribe with 16.13.1 node version', () => {
+      const dc = require('diagnostics_channel')
+      const ddTestChannel = dc.channel('dd:test:unsubscribe')
+
+      const version = process.version
+      if (version === 'v16.13.1') {
```

Comment:
This test is only here to probe that tests are running in a bugged node version. I plan to delete it if PR is accepted.